### PR TITLE
fix: snap sector status

### LINKED ISSUE #155: Fix sector state lmrpc return for Snap deal
Right now the method SectorState is aligned with PoRep pipeline. A snap deal results in us sending "Proving" state too early. This results is Boost trying to index the deal where unsealed copy doesn't exists yet.

### PR DESCRIPTION
**Pull request by LexLuthr**
Fixes #155

### CODE REVIEW

File: market/fakelm/lmimpl.go [hunk 1]
```diff
@@ -76,15 +76,19 @@ func (l *LMRPCProvider) SectorsStatus(ctx context.Context, sid abi.SectorNumber,
...
+		Complete        bool    `db:"after_commit_msg_success"`
+		Failed          bool    `db:"failed"`
+		SDR             bool    `db:"after_sdr"`
+		PoRep           bool    `db:"after_porep"`
+		Tree            bool    `db:"after_tree_r"`
+		IsSnap          bool    `db:"is_snap"`
+		Encode          bool    `db:"after_encode"`
+		SnapProve       bool    `db:"after_prove"`
+		SnapCommit      bool    `db:"after_prove_msg_success"`
+		SnapMoveStorage bool    `db:"after_move_storage"`
```
Comment (magik6k) [inline market/fakelm/lmimpl.go:91]: Don't those appear as null when the sector is not in the snap pipeline?

**Review by magik6k — COMMENTED** [review]
If this works then seems fine, but definitely needs to be tested manually at least

File: market/fakelm/lmimpl.go (hunk 1 above)
Comment (LexLuthr) [reply market/fakelm/lmimpl.go:91]: Thus COALESCE with FALSE so it can either be true or false.

**Comment by LexLuthr** [comment]
Closed in favor of #165

**Pull request closed without merging**